### DISCUSSION TITLE
would you consider supporting more commits conventions ?

### DISCUSSION THREAD

**Discussion by Roms1383** [ydcjeff/chlog#10] [Ideas]
Hi, I'm currently searching for a tool to generate changelog and ran into your tool, which looks hot !
I was wondering if you would consider supporting an additional commits convention : [gitmoji](https://gitmoji.dev/).
Please let me know, thanks :)

**Comment by jeffydc** [comment]
So far I have only used conventional commits, and I currently don't have any bandwidth for additional commit conventions.
